### PR TITLE
Feature landing page

### PR DESCRIPTION
**Pull request by fowobi**
create landing page using route to link it to App.js ,login button, and login form

### CODE REVIEW

File: client/src/components/LoginForm.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import  "../styles/login.css";
+import LoginButton from "./LoginButton";
+
```
Comment (carrejoe3) [inline client/src/components/LoginForm.jsx:3]: You don't need more than one line of whitespace between code blocks

File: client/src/styles/login.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  padding: 0;
+  /* user-select: none; */
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+html,
+body {
+  height: 100%;
+}
+/* body{
```
Comment (carrejoe3) [inline client/src/styles/login.css:13]: Do you still need this commented out code? It's good practice to not keep code as comments

File: client/src/styles/login.css
Comment (carrejoe3) [inline client/src/styles/login.css:1]: Need to remove the extra whitespace in the styling blocks, just to keep things tidy

File: client/src/styles/login.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  place-items: center;
+  background: #dde1e7;
+  text-align: center;
+  align-items: center;
+} */
+.login-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
```
Comment (carrejoe3) [inline client/src/styles/login.css:24]:
```suggestion
    height: calc(100vh - 43px);
```

This will stop the scrollbar from appearing on the login page

**Review by carrejoe3 — CHANGES_REQUESTED** [review]
Well done, the login page looks great! I've only left a few comments

File: client/src/components/LoginForm.jsx (hunk 1 above)
Comment (fowobi) [reply client/src/components/LoginForm.jsx:3]: Thank you for the feed back, I have corrected it all.

File: client/src/styles/login.css (hunk 3 above)
Comment (fowobi) [reply client/src/styles/login.css:24]: Corrected.

**Pull request merged**